llvm/test/CodeGen/X86/isel-br.ll

```diff
@@ -0,0 +1,31 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -O0 -mtriple=i686-linux-gnu                                     -verify-machineinstrs | FileCheck %s --check-prefix=DAG
```

Comment:
don't really need all the -verify-machineinstrs

Reply:
Do you mean for `isel-br.ll` file?